src/zocalo/configuration/__init__.py

```diff
@@ -369,7 +369,10 @@ def _merge_configuration(
     # Resolve all lazy file references relative to the specified context parameter
     for plugin in parsed:
         if isinstance(parsed[plugin], str):
-            parsed[plugin] = context.joinpath(parsed[plugin]).resolve()
+            parsed_plugin = (
+                context / pathlib.Path(parsed[plugin]).expanduser()
```

Comment:
any particular reason why not `parsed[plugin] = context.joinpath(parsed[plugin]).expanduser().resolve()` ?

I guess it might look a bit weird post-black.

Reply:
Understood it now.  (Need to `.expanduser()` on the path starting with `~` otherwise it'll just become a path part that is no longer resolvable)

Still, unnecessary to declare it as an extra variable?